### ISSUE TITLE
A lot of unit tests are failing

### ISSUE THREAD

**Issue by jespino** [OpenMOOC/moocng#49]
A lot of the unit tests are failing. I think a lof are for changes in the MongoDB documents structure without test upgrade, and others comes from race conditions in unit test because there aren't atomic.

**Comment by pitbulk** [comment]
You are right, we know that the unit tests of the API are failing. There were changes on the MongoDB scheme and many test has no sense anymore. We will rewrite the tests after we finish the "score functionality" implementation that will bring more changes to the MongoDB scheme.

After that test upgrade we also plan to update the tastypie version to a newer one.
